2025-03-10 09:24:48,349 - nisto - INFO - Starting Network Topology Designer
2025-03-10 09:24:48,814 - nisto - ERROR - Unhandled exception in main
Traceback (most recent call last):
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/main.py", line 47, in <module>
    sys.exit(main())
             ^^^^^^
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/main.py", line 40, in main
    window = MainWindow()
             ^^^^^^^^^^^^
  File "/home/cam/Desktop/NISTO/network-topology-designer/src/controllers/main_window.py", line 57, in __init__
    self.ui.graphicsView.setScene(self.scene)
    ^^^^^^^
AttributeError: 'MainWindow' object has no attribute 'ui'
